src/GovUk.Education.ExploreEducationStatistics.Data.Model/ReleaseFastTrack.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Azure.Cosmos.Table;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model
+{
+    public class ReleaseFastTrack : TableEntity
```

Comment:
For the work with table highlights (or 'top tables'), we should be able to store that information in this table entity to make it easier to query for them right?

Reply:
Yes, if it wants to be in the subject meta then the first issue we will face is no connection with the Content db. This suggestion would avoid that.